src/time_unix.c

```diff
@@ -30,7 +30,11 @@ extern "C"
 #include <math.h>
 
 #if defined(__ZEPHYR__)
+#if KERNEL_VERSION_MAJOR >= 3
```

Comment:
I might be mistaken, but this should be v3.1.0 or later? see https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v3.1.0

this could break with `v3.0.0`? it would be better to check minor kernel version too?

Reply:
Oh, you are right, If I'm not wrong it should be backward compatible to version 3.0.0, but it seems the best way is to check against the exact version.

Will update that.